### ISSUE TITLE
Accessibility Problem

### ISSUE THREAD

**Issue by agung2001** [artistudioxyz/bingopress#2]
This are a feedback from WP theme reviewer :

- [x] I am still unable to open sub-menus using keyboard navigation.
  - [x] Please check by creating a primary menu and make sure at least one of the top level items has sub-menu items.
  - [x] View the frontend and notice that using keyboard navigation the focus moves from the top level item with the sub-menu to the next top level item, without allowing you to open the dropdown sub-menu
  - [x] Once the sub-menu is open, the focus needs to be contained within the sub-menu allowing access to all the sub-menu items, and then the user should either be able to close the sub-menu or tab from the last sub-menu item to the next top level item.

- [x] Please also check the responsive issue by starting in desktop view, then narrow the viewport until the mobile navigation comes in to view. Open the mobile menu and then close it. Now widen the viewport until the mobile menu disappears and notice how the desktop menu is hidden, and the header layout is broken.
- [x] The issue is exactly the same if opening and closing the search modal instead of the mobile menu, and is a real world issue as rotating a device between portrait and landscape will trigger this problem.

**Comment by agung2001** [comment]
![opening-submenu-via-keyboard-navigation](https://user-images.githubusercontent.com/4479106/210102792-59e8d516-653e-41a8-af30-638bdf7c960e.gif)

**Comment by agung2001** [comment]
![responsive-issue-viewport-change](https://user-images.githubusercontent.com/4479106/210103324-4c0c0e88-424a-4b04-b8f5-b1e1aed7c9fa.gif)

**Comment by agung2001** [comment]
![responsive-issue-device-rotation](https://user-images.githubusercontent.com/4479106/210103357-e3b39e5d-e8a6-4a86-b7af-1edb4946f9a2.gif)

**Comment by agung2001** [comment]
The correct way is that : 
- [x] after opening the mobile menu, the focus should be trapped inside the mobile menu 
- [x] allowing the user to tab through the items in the correct order
- [x] then the last tab after the last menu item should take the focus to the menu close icon.

**Comment by agung2001** [comment]
![Mobile Keyboard Navigation](https://user-images.githubusercontent.com/4479106/210174623-7421b045-27a6-49c8-96bf-8c099477027c.gif)

**Comment by agung2001** [comment]
![Desktop Keyboard Navigation](https://user-images.githubusercontent.com/4479106/210174634-3371ea5d-340b-4ca2-baa3-ca24226afa95.gif)

**Comment by agung2001** [comment]
All tests are passed, closing the issue 👍
